ads/README.md

```diff
@@ -44,30 +44,31 @@ The ad itself is hosted within a document that has an origin different from the
 ### Available information
 We will provide the following information to the ad:
 
```

Comment:
thanks for re-org the section!
one small thing: shall we move `document.referrer` into the `window.context.referrer` bullet point, i read it as a clarification comparing to  `window.context.referrer`? and also, it is not really a part of context API.